internals/core_enums.py

```diff
@@ -269,45 +269,45 @@
         (STAGE_BLINK_DEV_TRIAL, []),
         (STAGE_BLINK_EVAL_READINESS, []),
         (STAGE_BLINK_ORIGIN_TRIAL,
-         [#GATE_DEBUGGABILITY_ORIGIN_TRIAL,
+         [GATE_DEBUGGABILITY_ORIGIN_TRIAL,
           GATE_API_ORIGIN_TRIAL]),
         (STAGE_BLINK_EXTEND_ORIGIN_TRIAL, [GATE_API_EXTEND_ORIGIN_TRIAL]),
         (STAGE_BLINK_SHIPPING,
          [
-          #GATE_PRIVACY_SHIP, GATE_SECURITY_SHIP, GATE_ENTERPRISE_SHIP,
-          #GATE_DEBUGGABILITY_SHIP, GATE_TESTING_SHIP,
+          GATE_PRIVACY_SHIP, GATE_SECURITY_SHIP, GATE_ENTERPRISE_SHIP,
+          GATE_DEBUGGABILITY_SHIP, GATE_TESTING_SHIP,
```

Comment:
It looks like all of these gates are created for every feature type. Is it confirmed that we want to create each of these gates for every feature upon creation?

Reply:
Good point.  I'll remove the gates on the FEATURE_TYPE_CODE_CHANGE_ID case.